.gitignore

```diff
@@ -27,3 +27,4 @@ main
 # CI
 .ci/
 build/
+.vscode/
```

Comment:
this should go in the user global .gitignore

Reply:
ok, done